raster/r.geomorphon/r.geomorphon.html

```diff
@@ -95,6 +95,7 @@ <h3>Forms represented by geomorphons:</h3>
 <DD>returns proportion between sides of the bounding box rectangle calculated for geomorphon rotated to fit least square line.</DD>
 <DT><b>width</b></DT>
 <DD>returns length of the shorter side of the bounding box rectangle calculated for geomorphon rotated to fit least square line.</DD>
+</DL>
```

Comment:
Only a minor issue: Does this fix also `document type does not allow element "H2" here; assuming missing "DD" start-tag`? I see it in the commit message, but I don't understand what is the change. Just trying to avoid missing it.

Reply:
Yes, with these changes the final HTML file passes the validation. The H2 element error was a consequence of the missing closing DL element.

Reply:
...or rather the closing tag of the DL element, for that matter.

Reply:
Thanks for clarifying it!